lib/l10n/intl_en.arb

```diff
@@ -214,5 +214,7 @@
   "stringForum": "cs.curs.pub.ro",
   "stringAnonymous": "Anonymous",
 
+  "newsFeedTitle": "News feed",
```

Comment:
Call this `navigationNewsFeed` and move it to the navigation group. I try to keep things grouped by type so it's easier to follow.